modules/olm-operatorhub-architecture.adoc

```diff
@@ -15,36 +21,66 @@ interface, it is only used directly by those who are creating their own
 Operators.
 ====
 
-[id="olm-operatorhub-arch-operatorsource_{context}"]
-== OperatorSource
-
-For each Operator, the OperatorSource is used to define the external data store
-used to store Operator bundles. A
-link:https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml[simple OperatorSource]
-includes:
-
-[cols="2a,8a",options="header"]
-|===
+[NOTE]
+====
+While OperatorHub no longer uses CatalogSourceConfig resources, they are still
+supported in {product-title}.
+====
 
-|Field
-|Description
+[id="olm-operatorhub-arch-operatorhub_crd_{context}"]
+== OperatorHub CRD
 
-|`type`
-|To identify the data store as an application registry, `type` is set to `appregistry`.
+The OperatorHub CRD can be used to change the state of the default
+OperatorSources provided with OperatorHub on the cluster from enabled to
+disabled and vice versa. This is useful when configuring {product-title} in
+restricted network environments:
 
-|`endpoint`
-|Currently, Quay is the external data store used by the OperatorHub, so
-the endpoint is set to `https://quay.io/cnr` for the Quay.io `appregistry`.
+.OperatorHub Custom Resource
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: OperatorHub
+metadata:
+  name: cluster
+spec:
+  disableAllDefaultSources: true <1>
+  sources: [ <2>
+    {
+      name: "community-operators",
+      disabled: false,
+    }
+  ]
+----
+<1> `disableAllDefaultSources` controls availability of all default OperatorSources
+that are configured by default during a {product-title} installation.
+<2> Default OperatorSources can be enabled or disabled individually using the `sources`
+section.
```

Comment:
The only other thing I would call out here is that `disableAllDefaultSources` is an override. So each individual default can be set to `disabled` if you set `disableAllDefaultSources` it will even disable the ones that have `disabled: false` set.

That sort of means that `Default OperatorSources can be enabled or disabled individually using the `sources` section` isn't really true. `disabled: false` doesn't mean "enabled", it just means "not explicitly disabled". Does that make sense?